src/room.cc

```diff
@@ -204,7 +204,7 @@ void np1secRoom::receive_handler(np1secMessage received_message)
                 delete message_session->second;
                 session_universe.erase(message_session->first);
               }
-              session_universe.insert(pair<string, np1secSession*>(received_message.session_id.get_as_stringbuff(), new_session));
+              session_universe.emplace(std::pair<std::string, np1secSession*>(received_message.session_id.get_as_stringbuff(), new_session));
```

Comment:
So I don't think emplace is safe here, that's why I changed it to insert. but because you have favored Yawning in three way merge confilct resolution, you reversed the change.